tests/test_hessian.py

```diff
@@ -881,13 +881,13 @@ def test_partial_water_num_hess():
 def test_numerical_hessian_in_daemon():
     """
     Ensure that no exceptions are raised when a numerical hessian is
-    calculated within a multiprocessing pool
+    calculated within a Joblib process pool
     """
-
-    with mp.pool.Pool(processes=1) as pool:
-
-        res = pool.apply_async(func=_calc_num_hessian_h2)
-        _ = res.get(timeout=None)
+    with Parallel(n_jobs=1) as parallel:
+        res = parallel(delayed(_calc_num_hessian_h2))
+    # What is the purpose of this function? I am not quite sure, so I replaced
+    # multiprocessing with joblib, but it does not seem to be necessary in joblib
+    # as joblib's Parallel pool does not spawn daemon processes.
```

Comment:
This was here because if you're optimising several things in parallel with an autode optimiser, that uses a numerical XTB hessian which executed using a multiprocessing (mp) pool. Does joblib support this kind of nested parallisum out of the box? what happens if you call a joblib parallel from within a spawn or fork-ed mp pool?